### PR TITLE
Mulebot and Door crush logging; Proxycheck note

### PR DESCRIPTION
**Pull request by Carbonhell**
Adds logging to mulebot runover proc and Door/shutters crush proc. Also makes proxycheck add a note to a ckey if it's positive to proxy using.

### CODE REVIEW

File: code/game/machinery/bots/mulebot.dm [hunk 1]
```diff
@@ -782,6 +782,7 @@ var/global/mulebot_count = 0
 	B.add_blood_list(H)
 	add_blood_list(H)
 	bloodiness += 4
+	add_logs(H, src, "ran over")
```
Comment (JohnGinnane) [inline code/game/machinery/bots/mulebot.dm]: I think maybe pass in src as object to add_logs for more robust logs perhaps

File: code/game/machinery/doors/door.dm [hunk 2]
```diff
@@ -255,6 +255,7 @@
 		else //for simple_animals & borgs
 			L.adjustBruteLoss(DOOR_CRUSH_DAMAGE)
 			playsound(L.loc, 'sound/effects/bang.ogg', 60, 1)
+		add_logs(L, src, "crushed")
```
Comment (JohnGinnane) [inline code/game/machinery/doors/door.dm]: Likewise, passing in the door as the object could possibly help

File: code/game/machinery/bots/mulebot.dm (hunk 1 above)
Comment (Carbonhell) [inline code/game/machinery/bots/mulebot.dm]: yer blind mate

Comment (JohnGinnane) [inline code/game/machinery/bots/mulebot.dm]: Yeah I'm not a smart man. Is it possible to get the user who's riding the mulebot then?

**Comment by Carbonhell** [comment]
dnm something went wrong

**Comment by Carbonhell** [comment]
[23:00:24] Has been ran over by яMulebot (#2) (NEWHP: 100)(82,117,1) = null
should be good to go(i've put add logs at the end of the proc,after the damage being done. I got no idea why newhp doesn't update.)

**Comment by JohnGinnane** [comment]
"я�"?

**Comment by Carbonhell** [comment]
àùè+ò cyka blyat what are you saying

**Comment by Carbonhell** [comment]
OH YOU MEANT THAT yeah it's due to mulebots having \mproper in their name,nothing major

**Comment by Crystalwarrior** [comment]
Everything seems good here, merging

**Pull request merged**